src/components/cadShippingMethod.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-nested-ternary */
+import React from 'react';
```

Comment:
nit: is it absolutely necessary to have 2 components for shipping LOL maybe we can just make the price a dynamic component next time, this is fine for now